typings/index.d.ts

```diff
@@ -380,7 +399,7 @@ declare module 'discord.js' {
     public valueOf(): N;
     public [Symbol.iterator](): IterableIterator<S>;
     public static FLAGS: unknown;
-    public static resolve(bit?: BitFieldResolvable<S, N>): number | bigint;
+    public static resolve(bit?: BitFieldResolvable<any, number | bigint>): number | bigint;
```

Comment:
This change was introduced in #5910 but needs to be reverted until a better solution can be found (not in this PR) to fix typings fully before moving forward.